pyzeebe/grpc_internals/grpc_channel_utils.py

```diff
@@ -21,10 +22,27 @@ def create_connection_uri(
 def create_channel(
     connection_uri: str,
     credentials: Optional[BaseCredentials] = None,
-    secure_connection: bool = False
+    secure_connection: bool = False,
+    options: Dict[str, Any] = None
 ) -> grpc.aio.Channel:
+    """
+    Create a gRPC channel.
+
+    Args:
+        connection_uri (str): The URI to Zeebe.
+        credentials (BaseCredentials): Credentials for accessing the channel. Default: None
+        secure_connection (bool): Create a secure channel (set to True when using credentials). Default: False
+        options (Dict[str, Any]): A key/value representation of `gRPC channel arguments_`. Default: None (will use library defaults)
+
+    Returns:
+        grpc.aio.Channel: A channel object set up to connect to Zeebe
+
```

Comment:
These are unnecessary. Maybe the explanation for `options` with the link could be nice

Reply:
If the user will create their own channel, I think offering this function could be an ok abstraction to have available, that's why I documented it this way. 

It does conflict a bit with the "internal" part of the module name though -  `from pyzeebe.grpc_internals.grcp_channel_utils import create_channel` doesn't sound like a public API. 

But OK, I will remove it for now, should we want to offer this in some form, we can always add that later.